assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -130,12 +130,14 @@ var ajaxCallbacks = {
             .text('Remove')
             .attr('data-remove-collaborator-link', email)
             .addClass('button button--link button--small flush hard--right');
+
         var $info = $element.find('.js-info');
         var $message = $info.find('.alert__message');
         var emailMessage = 'We have sent an email to <strong>'+ email +'</strong>';
+        var applicationTitle = $("#application").data("title");
```

Comment:
We avoid using ids for styling and/or JS as they don't allow things to be build in a reusable way. They're used for hooking into the DOM in acceptance tests instead.

If it's the only place where a title can be defined then maybe select it based on the existence of the data attribute? `$("[data-title]").data("title")`?